Source/WebCore/css/mediaControls.css

```diff
@@ -27,7 +27,8 @@
 @namespace "http://www.w3.org/1999/xhtml";
 
 /* media controls */
-::-webkit-media-controls {
+audio::-webkit-media-controls,
+video::-webkit-media-controls {
```

Comment:
`:is(audio, video)::-webkit-media-controls` also works, although I don't see much point in changing this codepath in general as I believe it's unused? cc @graouts @jernoble

Reply:
I opted for this based on @anttijk feedback on another bug's [comment](https://bugs.webkit.org/show_bug.cgi?id=268481#c2) for another pseudo-element, so I just thought to restrict this and also noticed that Blink also do it.

Reply:
I can confirm this codepath is unused aside from testing.

Reply:
I was checking today to see - if we can remove but it seems `Playstation` port list this in their [PlatformPlayStation.cmake](https://searchfox.org/wubkat/source/Source/WebCore/PlatformPlayStation.cmake) and [Adwaita.cmake](https://searchfox.org/wubkat/source/Source/WebCore/platform/Adwaita.cmake#30). So I think meanwhile it would be good to land. I will ask them separately, if we can remove this.